image_tools/CMakeLists.txt

```diff
@@ -45,33 +45,18 @@ if(BUILD_TESTING)
   find_package(ament_lint_auto REQUIRED)
   ament_lint_auto_find_test_dependencies()
 
+  # get the rmw implementations ahead of time
   find_package(rmw_implementation_cmake REQUIRED)
+  get_available_rmw_implementations(rmw_implementations)
+  foreach(rmw_implementation ${rmw_implementations})
+    find_package("${rmw_implementation}" REQUIRED)
+  endforeach()
```

Comment:
OK I see now that the only info that's needed is the list of RMW implementations from `get_all_rmw_implementations`, not the packages themselves. I will remove this block thanks.